src/content/recipes/next.md

```diff
@@ -0,0 +1,84 @@
+<div class="aside aside__no-top">
+
+This recipe assumes that you are using Next.js >= 12.x as well as Storybook >= 7.x
+
+</div>
+
+<RecipeHeader>
+
+How to setup Next.js and Storybook
+
+</RecipeHeader>
+
+Next.js is used in tens of thousands of websites and apps. With the recent release of v13, it’s more capable than ever, including many improvements to key features like routing and image optimization. But with all this change, it can be tough to migrate your existing Next app, much less everything it integrates with.
```

Comment:
Can we run this through ChatGPT to reword? Concerned about SEO with 100% duplicate content.

Reply:
Sure! Good idea